src/main/java/org/team1540/robot2018/Tuning.java

```diff
@@ -1,4 +1,201 @@
+
+  // @Tunable("Elevator Exchange Position")
+  public static double elevatorExchangePosition = 500;
+
+  // @Tunable("Elevator Front Switch Position")
+  public static double elevatorFrontSwitchPosition = 2900;
+
+  // @Tunable("Elevator Scale Lower Position")
+  public static double elevatorScalePosition = 7400;
+
+  // @Tunable("Elevator Low Scale Position")
+  public static double elevatorLowScalePosition = 6200;
+
+  // @Tunable("Elevator Obstacle Position")
+  public static double elevatorObstaclePosition = 1300;
+
+  // @Tunable("Elevator Obstacle Upper Position")
+  public static double elevatorObstacleUpperPosition = 3750;
+
+  @Tunable("Max Elevator Deviation")
+  public static double maxElevatorDeviation = 200;
+
+  // WRIST
+  @Tunable("Wrist P")
+  public static double wristP = 10.0;
+
+  @Tunable("Wrist I")
+  public static double wristI = 0;
+
+  @Tunable("Wrist D")
+  public static double wristD = 0;
+
+  @Tunable("Wrist F")
+  public static double wristF = 1.364;
+
+  @Tunable("Wrist I Zone")
+  public static int wristIzone = 0;
+
+  @Tunable("Motion Cruise Velocity")
+  public static int wristCruiseVelocity = 600;
+
+  @Tunable("WRist Max Acceleration")
+  public static int wristMaxAccel = 1000;
+  @Tunable("Wrist Peak Current Limit")
+  public static int wristCurrentLimit;
+  @Tunable("Wrist Peak Duration")
+  public static int wristPeakDuration;
+
+  @Tunable("Wrist Stop Tolerance")
+  public static double wristTolerance = 50;
+
+  // @Tunable("Wrist Out Position")
+  public static double wristOutPosition = 8250;
+  // @Tunable("Wrist Back Position")
+  public static double wristBackPosition = 0;
+  // @Tunable("Wrist Transit Position")
+  public static double wristTransitPosition = 3900;
+  // @Tunable("Wrist 45 Back Position")
+  public static double wrist45BackPosition = 1500;
+  // @Tunable("Wrist 45 Forward Position")
+  public static double wrist45FwdPosition = 6200;
+
+  @Tunable("Max wrist deviation")
+  public static double maxWristDeviation = 200;
+
+  // WINCH
+  @Tunable("Winch In Speed")
+  public static double winchInSpeed = 1;
+
+  @Tunable("Winch Out Speed")
+  public static double winchOutSpeed = 1; // 0.75
+
+  // TAPE
+  @Tunable("Tape In Speed")
+  public static double tapeInSpeed = .5;
+
+  @Tunable("Tape Out Speed")
+  public static double tapeOutSpeed = -.5;
+
+  @Tunable("Tape Measure Multiplier")
+  public static double tapeMeasureMultiplier = 1;
+
+  @Tunable("Winch Multiplier")
+  public static double winchMultiplier = 1;
+
+  @Tunable("Climber In Speed")
+  public static double climberInSpeed = 0.5;
+
+  @Tunable("Climber Out Speed")
+  public static double climberOutSpeed = -0.4;
+
+  @Tunable("Manual Superstructure Control Deadzone")
+  public static double manualControlDeadzone = 0.2;
+  @Tunable("Climber In Low Speed")
+  public static double climberInLowSpeed = -0.4;
+  @Tunable("Climber In HIgh Speed")
+  public static double climberInHighSpeed = -1;
```

Comment:
all the climber constants in Tuning need to be cleaned up. check which constants are actually getting used, because a lot of them need to be deleted, and also they should all be in the same place.